apidoc/Titanium/App/iOS/iOS.yml

```diff
@@ -1217,6 +1217,10 @@ properties:
     type: Dictionary
     optional: true
     since: "5.4.0"
+    
+  - name: inBackground
```

Comment:
This should be written inside  
events : 
    "localnotificationaction" section because it tells the about received data and "inBackground" is property of received local notification data. Currently it written inside " NotificationParams", which is local notification creation object.